packages/core/src/bundle.ts

```diff
@@ -224,6 +235,10 @@ function makeBundleVisitor(
           return;
         }
 
+        if (keepUrlRefs && isUrlRef(node.$ref)) {
```

Comment:
Could you use `isAbsoluteUrl` from **packages/core/src/ref-utils.ts**? You may probably adjust it to be case insensitive.

Reply:
Thanks for pointing me to this function. Question: it seems you assume lower case `http(s)://` prefixes for absolute urls in your codebase (also in the `isMappingRef` in **packages/core/src/ref-utils.ts**). I'm fine keeping the `isAbsoluteRef` as is and dropping the case-insensitive check for consistency. Wdyt?

Reply:
I think it should be case insensitive all over the codebase, but I'd rather do this a separate merge request.

Reply:
If we open an issue, please be aware: Only the scheme and host are case insensitive. The remaining part (paths, parameters, etc) of the URL are case sensitive.

https://www.rfc-editor.org/rfc/rfc3986

> [6.2.2.1](https://www.rfc-editor.org/rfc/rfc3986#section-6.2.2.1).  Case Normalization
> 
>    For all URIs, the hexadecimal digits within a percent-encoding
>    triplet (e.g., "%3a" versus "%3A") are case-insensitive and therefore
>    should be normalized to use uppercase letters for the digits A-F.
> 
>    When a URI uses components of the generic syntax, the component
>    syntax equivalence rules always apply; namely, that the scheme and
>    host are case-insensitive and therefore should be normalized to
>    lowercase.  For example, the URI <HTTP://www.EXAMPLE.com/> is
>    equivalent to <http://www.example.com/>.  The other generic syntax
>    components are assumed to be case-sensitive unless specifically
>    defined otherwise by the scheme (see [Section 6.2.3](https://www.rfc-editor.org/rfc/rfc3986#section-6.2.3)).
>